11/umbraco-cms/reference/querying/ipublishedcontent/properties.md

```diff
@@ -230,6 +230,11 @@ var mediaItems = Model.Value<IEnumerable<IPublishedContent>>(PublishedValueFallb
 
 If the current content item doesn't have the requested value, use an alternative 'fallback' value in its place.
 
+Each of the examples below make use of an injected PublishedValueFallback. This is achieved by adding the following at the top of your Razor file:
+
+@inject IPublishedValueFallback PublishedValueFallback
```

Comment:
```suggestion
```csharp
@inject IPublishedValueFallback PublishedValueFallback
```
```

Reply:
Not sure why it doesn't add the end tags for this codeblock element.
Might clash with the "suggestions" formats here on GitHub 😅